protocol/v1/membership_update_message.go

```diff
@@ -86,13 +98,18 @@ func MembershipUpdateMessageFromProtobuf(raw *protobuf.MembershipUpdateMessage)
 	return &MembershipUpdateMessage{
 		ChatID:  raw.ChatId,
 		Events:  events,
-		Message: raw.Message,
+		Message: raw.GetMessage(),
```

Comment:
Need to also handle an EmojiReaction field

Reply:
@cammellos am I mistaken in thinking that this section needs to also handle `EmojiReaction`s?

Something like:

```
Message: raw.GetMessage(),
EmojiReaction: raw.GetEmojiReaction(),
```

Reply:
oh good point, I missed that, thanks!